docs/intro.rst

```diff
@@ -415,14 +411,6 @@ This is the current list of error and warning codes:
 +------------+----------------------------------------------------------------------+
 | **W6**     | *Deprecation warning*                                                |
 +------------+----------------------------------------------------------------------+
-| W601       | .has_key() is deprecated, use 'in'                                   |
-+------------+----------------------------------------------------------------------+
-| W602       | deprecated form of raising exception                                 |
-+------------+----------------------------------------------------------------------+
-| W603       | '<>' is deprecated, use '!='                                         |
-+------------+----------------------------------------------------------------------+
-| W604       | backticks are deprecated, use 'repr()'                               |
```

Comment:
Perhaps it should still be mentioned in the documentation that these 4 codes have been in use before and should not be used again for future checks?

Reply:
I would just remove them as you've done